docs/src/running-tests-python.md

```diff
@@ -47,6 +47,14 @@ You can run a single test, a set of tests or all tests. Tests can be run on one
   pytest test_login.py --browser webkit --browser firefox
   ```
 
+- Running Tests in parallel
+
+  ```bash
+  pytest --numprocesses auto
+  ```
+
+  (This assumes `pytest-xdist` is installed. For more information see [here](./test-runners.md#parallelism-running-multiple-tests-at-once).)
+
```

Comment:
@mxschmitt wdyt about adding this section?

Since it's a drop-in speed up, I like making sure users will find it, so having it here too is nice.

Reply:
sgtm